splunk_topology/check.py

```diff
@@ -126,8 +129,18 @@ def check(self, instance):
 
     def _dispatch_and_await_search(self, instance, saved_searches):
         start_time = time.time()
+
+        # don't dispatch if sids present
+        if self.status.data.get(instance.instance_config.base_url) is not None:
+            for (sid, saved_search) in self.status.data[instance.instance_config.base_url]:
+                instance.splunkHelper.finalize_sid(sid, saved_search)
+            self.status.data[instance.instance_config.base_url] = []
```

Comment:
can this reuse the extracted method `update_status` in the splunk_telemetry_base.py ?